src/common/Mutex.cc

```diff
@@ -92,6 +92,8 @@ Mutex::~Mutex() {
 void Mutex::Lock(bool no_lockdep) {
   int r;
 
+  r = 0;
```

Comment:
`r` is either initialized at https://github.com/ceph/ceph/pull/7659/files#diff-a405b020a455d3ae9e1c00c67eb8fd2cR107 or at https://github.com/ceph/ceph/pull/7659/files#diff-a405b020a455d3ae9e1c00c67eb8fd2cR112, i think. am i missing anything?